requirements-common.txt

```
@@ -20,7 +20,7 @@ tiktoken >= 0.6.0  # Required for DBRX tokenizer
 lm-format-enforcer >= 0.10.11, < 0.11
 outlines == 0.1.11
 lark == 1.2.2
-xgrammar == 0.1.11; platform_machine == "x86_64"
+xgrammar == 0.1.14
```

We have more CPUs supported than just x86 and ARM
```suggestion
xgrammar == 0.1.14; platform_machine == "x86_64" or platform_machine == "aarch64"
```